backend/fbdev.c

```diff
@@ -0,0 +1,315 @@
+/*
+ * Twin - A Tiny Window System
+ * Copyright (c) 2024 National Cheng Kung University, Taiwan
+ * All rights reserved.
+ */
+
+#include <fcntl.h>
+#include <linux/fb.h>
+#include <linux/kd.h>
+#include <linux/vt.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include <twin.h>
+#include <unistd.h>
+
+#include "linux_input.h"
+#include "twin_backend.h"
+
+#define VT_MAX 63
```

Comment:
Why 63?

Reply:
According to ChatGPT, the typical number of virtual terminals on GNU/Linux ranges from 0 to 63.
This is indeed the case on my Ubuntu machine by verifying with `ls /dev/tty*`.

However, I think we should set `VT_MAX` to 64 to ensure correct enumeration in the `twin_find_free_vt()` for-loop.

Reply:
> This is indeed the case on my Ubuntu machine by verifying with `ls /dev/tty*`.

No, specify fewer VTs.

Reply:
> > This is indeed the case on my Ubuntu machine by verifying with `ls /dev/tty*`.
> 
> No, specify fewer VTs.

Removed the line as it is no longer required.